sdk/cosmosdb/cosmos/src/client/Item/Item.ts

```diff
@@ -20,6 +20,7 @@ import { Resource } from "../Resource";
 import { ItemDefinition } from "./ItemDefinition";
 import { ItemResponse } from "./ItemResponse";
 import { getEmptyCosmosDiagnostics, withDiagnostics } from "../../utils/diagnostics";
+import { EncryptionProcessor } from "../../encryption";
```

Comment:
should we add `EncryptionProcessor` as a property in `Container`? This way we can directly use it in both `item` and `items`.